net/hass/files/hass.conf

```diff
@@ -0,0 +1,5 @@
+config hass 'global'
+        option host '1.2.3.4:8123'
```

Comment:
if you _must_ do this, use something.example.org instead of 1.2.3.4

Reply:
I think a local-only IP would be better than a DNS name that could be registered at some point, but yeah, 1.2.3.4 is a bad idea.

Reply:
No, I literally meant "something.example.org"  example.{com,org,net} are permanently reserved: https://tools.ietf.org/html/rfc2606  no-one can register them :)

Reply:
TIL, thanks!

Reply:
The TLD `example.` is also reserved, so I used that.